addons/html_editor/static/src/core/history_plugin.js

```diff
@@ -661,6 +667,10 @@ export class HistoryPlugin extends Plugin {
         // Reapply the uncommited draft, since this is not an operation which should cancel it
         this.applyMutations(this.currentStep.mutations);
         this.dispatch("ADD_EXTERNAL_STEP");
+        this.applyMutations(newStep.mutations);
+        this.dispatch("NORMALIZE", {
+            node: this.getMutationsRoot(newStep.mutations) || this.editable,
+        });
```

Comment:
@Mtaylorr Ah I'm sorry I was not clear enough the line `this.applyMutations(newStep.mutations);` already exist and is a little bit above (line 662), and I meant to tell you to move the normalize next to that line :)

Reply:
Ah yeah my bad , I knew it existed , I forgot to remove it after the copy paste